api/application.py

```diff
@@ -49,6 +52,39 @@ def sesam_run():
 
     return jsonify(output_payload)
 
+@application.route('/api/matlab', methods=['POST'])
+def matlab_test():
+    if not request.json:
+        abort(400)
+
+    with NamedTemporaryFile(delete=False, suffix='.json', mode='wt') as inf:
+        with NamedTemporaryFile(delete=False, suffix='.json') as outf:
```

Comment:
You can also add this to the `with` above to avoid extra indentation.

`with TempFile as inf, TempFile as outf:`